packages/expo-battery/android/src/main/java/expo/modules/battery/BatteryLevelReceiver.java

```diff
@@ -0,0 +1,24 @@
+package expo.modules.battery;
+
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+import android.content.IntentFilter;
+import android.os.BatteryManager;
+import android.os.PowerManager;
+
+public class BatteryLevelReceiver extends BroadcastReceiver {
+  public static String INTENT_ACTION = "expo.modules.battery.BatteryLevelReceiver";
```

Comment:
is this used?

Reply:
Yes. This is used for event listener in the battery module. This is triggered when battery level dropped below `"android.intent.action.BATTERY_LOW"` or up to `"android.intent.action.BATTERY_OKAY"` from low battery level.  I just used this `Intent.ACTION_BATTERY_CHANGED` intent filter to get the current battery level and return the value back.